src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -329,4 +332,5 @@ class Repository(
       }
 
   fun deleteAllNonAssociations() = offenderNonAssociationRepository.deleteAll()
+  fun delete(adjustment: OffenderSentenceAdjustment) = offenderSentenceAdjustmentRepository.delete(adjustment)
 }
```

Comment:
cascade delete is failing when sentence adjustment is linked to key date adjustment - not sure how you force order of deletes (sentences would need to be deleted before keydates)